sparkle/src/components/EmptyCTA.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+
+import { Button, ButtonProps } from "@sparkle/components/";
+import { cn } from "@sparkle/lib/utils";
+
+interface EmptyCTAProps extends React.HTMLAttributes<HTMLDivElement> {
+  action: React.ReactNode;
+  message: string;
+}
+
+const EmptyCTA = React.forwardRef<HTMLDivElement, EmptyCTAProps>(
```

Comment:
OOC, what will this used for?

Reply:
@Duncid can you shed some light here?

Reply:
Yes :)
It's a new component to standardize all of these:
<img width="977" alt="image" src="https://github.com/user-attachments/assets/87a97b5d-09ee-4e48-ae25-28394ce434e1">